apps/examples/src/hooks/usePerformance.ts

```diff
@@ -12,6 +12,7 @@ export function usePerformance(): TLUiOverrides {
 		actions(_editor, actions) {
 			Object.keys(actions).forEach((key) => {
 				const action = actions[key]
+				if (!action) return
```

Comment:
nit: can this just be: `const action = actions[key]!` ?

Reply:
the key could be present, but the value could be `undefined`